doc/api/fs.md

```diff
@@ -1524,11 +1524,27 @@ If `options` is a string, then it specifies the encoding. Example:
 ```js
 fs.readFile('/etc/passwd', 'utf8', callback);
 ```
+*Note*: When the given path identifies a directory, the behavior of
```

Comment:
Would it be too imprecise to say `When the path is a directory` instead of `When the given path identifies a directory`? The former seems clearer to me, even if the latter may be more precise.